package.json

```diff
@@ -11,6 +11,7 @@
     "@babel/core": "^7.4.0",
     "@wordpress/blocks": "^6.2.0",
     "@wordpress/browserslist-config": "^2.2.2",
+    "@wordpress/components": "^7.3.1",
```

Comment:
The only reason it's explicitly installed is because the unit tests need it, otherwise something like this happens:
```
 FAIL  assets/src/components/imageUpload/imageUpload.test.js
  ● Test suite failed to run

    Cannot find module '@wordpress/components' from 'index.js'

       7 |  */
       8 | import { Component } from '@wordpress/element';
    >  9 | import { Button } from '@wordpress/components';
         | ^
      10 | import { data } from '@wordpress/data';
      11 | 
      12 | /**

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:229:17)
      at Object.<anonymous> (assets/src/components/imageUpload/index.js:9:1)
```

Reply:
Ah, yup, I get the same thing. This problem rings a bell and I'm going to try to get to the bottom of it, but this definitely isn't a blocker—later on we can do a separate PR that trims the dependencies.